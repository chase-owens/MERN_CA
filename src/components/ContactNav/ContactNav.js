import React from 'react';
const logoPath = require('images/Logo.png');
import { isMobile } from 'react-device-detect';
import theme from 'styles/theme';

import Grid from '@material-ui/core/Grid';
import Button from '@material-ui/core/Button';
import { Link } from 'react-router-dom';

const ContactNav = () => {
  return (
    <nav>
      <Grid container justify='space-between'>
        <Grid
          item
          style={{
            marginTop: isMobile ? 25 : 5,
            marginLeft: 20,
            marginRight: isMobile ? 25 : 5
          }}
        >
          <img height={isMobile ? 100 : 50} src={logoPath} alt='logo' />
        </Grid>
        <Grid item style={{ margin: '10px 15px' }}>
          <Link style={{ textDecoration: 'none' }} to='/'>
            <Button style={{ fontSize: isMobile ? '2.1em' : null }}>
              Home
            </Button>
          </Link>
        </Grid>
      </Grid>
    </nav>
  );
};

export default ContactNav;
