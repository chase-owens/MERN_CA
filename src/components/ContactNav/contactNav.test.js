import React from 'react';
import ContactNav from './ContactNav';
import { shallow } from 'enzyme';

describe('ContactNav', () => {
  it('renders without crashing', () => {
    const contactNav = shallow(<ContactNav />);
    expect(contactNav).toMatchSnapshot();
  });
});
