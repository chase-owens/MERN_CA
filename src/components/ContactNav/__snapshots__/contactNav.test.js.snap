// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactNav renders without crashing 1`] = `
<nav>
  <WithStyles(Grid)
    container={true}
    justify="space-between"
  >
    <WithStyles(Grid)
      item={true}
      style={
        Object {
          "marginLeft": 20,
          "marginRight": 5,
          "marginTop": 5,
        }
      }
    >
      <img
        alt="logo"
        height={50}
        src={
          Object {
            "default": "",
          }
        }
      />
    </WithStyles(Grid)>
    <WithStyles(Grid)
      item={true}
      style={
        Object {
          "margin": "10px 15px",
        }
      }
    >
      <Link
        style={
          Object {
            "textDecoration": "none",
          }
        }
        to="/"
      >
        <WithStyles(Button)
          style={
            Object {
              "fontSize": null,
            }
          }
        >
          Home
        </WithStyles(Button)>
      </Link>
    </WithStyles(Grid)>
  </WithStyles(Grid)>
</nav>
`;
